class_object.rb

```diff
@@ -0,0 +1,62 @@
+
+class Student
+
+  #class variable 
+  @@no_of_students = 0
+  
+  #instance variable
+  @first_name
```

Comment:
@dinnupatel100 Why instance variable initialised here as well as in def initialize ?